src/drivers/mct/cime_config/config_component_cesm.xml

```diff
@@ -2,7 +2,31 @@
 
 <?xml-stylesheet type="text/xsl" href="config_compsets.xsl" ?>
 
-<entry_id>
+<entry_id version="3.0">
+  <!-- modifier_mode allowed values are
+       '*' 0 or more modifiers (default)
+       '1' exactly 1 modifier
+       '?' 0 or 1 modifiers
+       '+' 1 or more modifiers
+
+       This file may have forcing and cpl desc entries.
+  -->
+  <description modifier_mode="1">
+    <desc forcing="1850"></desc>
+    <desc forcing="2000">1972-2004</desc>
+    <desc forcing="2003">2002-2003</desc>
+    <desc forcing="HIST">Historic transient </desc>
+    <desc forcing="20TR">Twentieth century transient</desc>
+    <desc forcing="4804"></desc>
+    <desc forcing="RCP2">CMIP5 rcp 2.6 forcing</desc>
+    <desc forcing="RCP4">CMIP5 rcp 4.5 forcing</desc>
+    <desc forcing="RCP6">CMIP5 rcp 6.0 forcing</desc>
+    <desc forcing="RCP8">CMIP5 rcp 8.5 forcing</desc>
+    <desc cpl="BGC[%BDRD][%BPRP]">Biogeochemistry intercomponent </desc>
+    <desc option="BDRD"> with diagnostic CO2</desc>
+    <desc option="BPRP"> with prognostic CO2 </desc>
+
```

Comment:
Yes, as long as you don't change to the version="3.0" format things will work as they do now.